backend/src/index.ts

```diff
@@ -1,36 +1,28 @@
 import "module-alias/register";
-import express, { Router, Request, Response } from "express";
+import cors from "cors";
+import express from "express";
+import { initDatabase } from "./database";
+
 import router from "@/routers/index";
 import logging from "@/utils/logging";
-import { initDatabase } from "./database";
 import { initFirebase } from "@/configs/firebaseConfig";
-import { CLIENT_ORIGIN, SERVER_PORT } from "@/configs/config";
 import { errorHandler } from "@/middlewares/errorHandler";
-import cors from "cors";
+import { loggingHandler } from "@/middlewares/loggingHandler";
+import { CLIENT_ORIGIN, SERVER_PORT } from "@/configs/config";
 
 const app = express();
-app.use(cors({origin: CLIENT_ORIGIN}))
+app.use(cors({ origin: CLIENT_ORIGIN }));
```

Comment:
Make sure to have production and development origins otherwise, you will have CORS-related issues. Also, it's a good idea to address XSS-related issues as well. More read - https://owasp.org/www-project-top-ten/2017/A7_2017-Cross-Site_Scripting_(XSS).html